source/backend/dal/Repositories/PropertyActivityFileDocumentRepository.cs

```diff
@@ -0,0 +1,87 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Claims;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Logging;
+using Pims.Core.Extensions;
+using Pims.Dal.Entities;
+
+namespace Pims.Dal.Repositories
+{
+    /// <summary>
+    /// PropertyActivityDocumentRepository class, provides a service layer to interact with document acquisition files within the datasource.
+    /// </summary>
+    public class PropertyActivityDocumentRepository : BaseRepository<PimsPropertyActivityDocument>, IPropertyActivityDocumentRepository
+    {
+        #region Constructors
+
+        /// <summary>
+        /// Creates a new instance of a PropertyActivityDocumentRepository, and initializes it with the specified arguments.
+        /// </summary>
+        /// <param name="dbContext"></param>
+        /// <param name="user"></param>
+        /// <param name="logger"></param>
+        public PropertyActivityDocumentRepository(PimsContext dbContext, ClaimsPrincipal user, ILogger<AcquisitionFileDocumentRepository> logger)
+            : base(dbContext, user, logger)
+        {
+        }
+        #endregion
+
+        #region Methods
+
+        /// <summary>
+        /// Get a list of all the document file relationships for a a given acquisition file.
+        /// </summary>
+        /// <returns></returns>
+        public IList<PimsPropertyActivityDocument> GetAllByPropertyActivityFile(long fileId)
```

Comment:
variable and method naming looks copy pasted? 
fileId => activityId | propertyActivityId
GetAllByPropertyActivityFile => GetAllByPropertyActivity